test/common/quic/envoy_quic_dispatcher_test.cc

```diff
@@ -197,12 +218,17 @@ class EnvoyQuicDispatcherTest : public QuicMultiVersionTest,
                             const std::vector<Network::FilterFactoryCb>& filter_factories) {
           EXPECT_EQ(1u, filter_factories.size());
           Server::Configuration::FilterChainUtility::buildFilterChain(connection, filter_factories);
+          dynamic_cast<EnvoyQuicServerSession&>(connection)
+              .set_max_inbound_header_list_size(64 * 1024);
           return true;
         }));
     EXPECT_CALL(*read_filter, onNewConnection())
         // Stop iteration to avoid calling getRead/WriteBuffer().
         .WillOnce(Return(Network::FilterStatus::StopIteration));
-    EXPECT_CALL(network_connection_callbacks, onEvent(Network::ConnectionEvent::Connected));
+    if (!quicVersionUsesTls()) {
+      // 0-RTT is not supported in Quic TLS handshake yet.
```

Comment:
I think 0-RTT is supported by QUIC TLS, do you mean it is not supported by Envoy/QUIC, or it is not used in this test?

Reply:
The comment is outdated. The QUICHE test utils GenerateFullCHLO() doesn't support IETF CHLO. So this tests only expect handshake complete for Google QUIC.